src/pymorize/cmorizer.py

```diff
@@ -257,6 +260,25 @@ def _post_init_populate_rules_with_data_request_variables(self):
             self._rules_expand_drvs()
             self._rules_depluralize_drvs()
 
+    def _post_init_create_controlled_vocabularies(self):
+        """
+        Reads the controlled vocabularies from the directory tree rooted at
+        `<tables_dir>/CMIP6_CVs` and stores them in the `controlled_vocabularies`
```

Comment:
```suggestion
        ``<tables_dir>/CMIP6_CVs`` and stores them in the ``controlled_vocabularies``
```

If readthedocs uses rst, then what you are looking for is double quotes, single just make coursive.